sktime/annotation/adapters/_sktime_od.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+from sktime.annotation.base._base import BaseSeriesAnnotator
+import pandas as pd
+import numpy as np
+from sklearn import clone
+from sktime.forecasting.base import ForecastingHorizon
+
+__author__ = ["satya-pattnaik", "mloning"]
+
+
+class SktimeODAnnotator(BaseSeriesAnnotator):
+    """An Outlier Detector which detects those time points as outliers for which the values
+    fall outside of the Uncertainty Intervals of forecasts generated from
+    a Sktime Forecaster.
+    The Sktime Forecaster is passed as an argument.
+
+    This method is inspired from :
+    https://docs.seldon.io/projects/alibi-detect/en/latest/methods/prophet.html
+    Parameters
+    ----------
+    estimator : a Sktime Forecaster object
+    fmt : str {"dense", "sparse"}, optional (default="dense")
+        Annotation output format:
+        * If "sparse", a sub-series of labels for only the outliers in X is returned,
+        * If "dense", a series of labels for all values in X is returned.
+    labels : str {"indicator", "score"}, optional (default="indicator")
+        Annotation output labels:
+        * If "indicator", returned values are boolean, indicating whether a value is an
+        outlier,
+        * If "score", returned values are floats, giving the outlier score.
+    alpha : float, Significance level of the Uncertainty Interval to be used,
+            for detecting the Outliers,(default=0.05)
+    """
+
+    def __init__(self, estimator, fmt="dense", labels="indicator", alpha=0.05):
+        self.estimator = estimator  # Sktime forecaster
+        self.alpha = alpha
+        super(SktimeODAnnotator, self).__init__(fmt=fmt, labels=labels)
+
+    def _fit(self, X, Y=None):
+        """Fit to training data.
+
+        core logic
+
+        Parameters
+        ----------
+        X : pd.DataFrame
+            training data to fit model to, time series
+        Y : pd.Series, optional
+            ground truth annotations for training if annotator is supervised
+        Returns
+        -------
+        self : returns a reference to self
+
+        State change
+        ------------
+        creates fitted model (attributes ending in "_")
+        """
+        X_t = X.copy()
+        self.estimator_ = clone(self.estimator)
```

Comment:
We need to check that the forecaster can actually generate prediction intervals. I think @fkiraly added a tag for that, something like `capability:pred_int`. So we should check if `estimator.get_tag("capability:pred_int")` evaluates to True.

Reply:
I don't see this tag in some of the forecasters' code. May be you I am looking in the wrong place.
So do I need to write something like this?

```
if estimator.get_tag("capability:pred_int"):
    Execute the code
else:
    raise NotImplementedError

```

Reply:
Yes, or simply have a `_check_estimator` stand-alone function which raises the NotImplementedError. @fkiraly could you help with the tags here?

Reply:
<img width="566" alt="Screenshot 2021-09-01 at 5 31 44 PM" src="https://user-images.githubusercontent.com/22102468/131667921-54747632-1ac3-4f6d-a15d-480f955e0ce2.png">

This is what I get for ARIMA, I don't see anything for prediction(my version is ```0.7.0```). Or I am looking at the wrong thing? @mloning @fkiraly

Reply:
When I'm using the latest version from `main`, this gives me the tags as expected:
```python
from sktime.forecasting.arima import ARIMA

forecaster = ARIMA()
forecaster.get_tags()
```